index.html

```diff
@@ -61,9 +62,204 @@ <h3>LET'S CONNECT</h3>
                             </li>
                         </ul>
                     </div>
+               
+                    <!-- start work section -->
+                    <section>
+                    <div class="work-section">
```

Comment:
- [ ] As the previous code reviewer suggested. I kindly suggest that you use a `section` tag here instead of a `div` tag

### EXAMPLE
:x: Wrong
`<div class="work-section">
`

:heavy_check_mark: Right
`<section class="work-section">
`